README.md

```diff
@@ -44,14 +44,12 @@ $ mvn package
 
 ### Bundling a Phoenix Client
 
-To build a release of PQS which packages a specific version of Phoenix, enable the `package-phoenix-client` profile
-and specify properties to indicate a specific Phoenix version.
+To build a release of PQS which packages a specific version of Phoenix, specify the `package-phoenix-client` system property
+and specify the `phoenix.version` system property to indicate a specific Phoenix version.
 
-By default, PQS will package the same version of Phoenix used for build/test. This version is controlled by the system
-property `phoenix.version` system property. Depending on the version of Phoenix, you may also be required to
-use the `phoenix.hbase.classifier` system property to identify the correct version of Phoenix built against
-the version of HBase of your choosing.
+PQS will package the same version of Phoenix used for build/test. This version is controlled by the
+ `phoenix.version` system property.
```

Comment:
If we don't have activation for all profiles off of phoenix.version values, we should call out the need to explicitly provide the profile.

Reply:
It's a good idea, that way we document how to built with future/custom versions.